Woof/Woof/Helper/AppAlert/AppAlert.swift

```diff
@@ -0,0 +1,8 @@
+/// Represents the alerts that are used in the application.
+enum AppAlert: String, CaseIterable {
```

Comment:
Alerts have a title and a message. Will this enum cover both cases? And as discussed earlier do we really need the enum with the restricted cases?

Reply:
Agreed I replaced cases with static properties.